cluster/gce/config-default.sh

```diff
@@ -546,3 +546,10 @@ PROXY_TAG="${SCALEOUT_PROXY_NAME:-scaleoutproxy}"
 KUBERNETES_RESOURCE_PARTITION="${KUBERNETES_RESOURCE_PARTITION:-false}"
 # true if this is a tenant partition cluster
 KUBERNETES_TENANT_PARTITION="${KUBERNETES_TENANT_PARTITION:-false}"
+
+if [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == true ]]; then
```

Comment:
Maybe "if KUBERNETES_RESOURCE_PARTITION==true && ENABLE_INSECURE_SCALEOUT_CLUSTER_MODE==true"

Reply:
action depends on the decision in line 99 on how we can address Prometheus etc to connect to secured api server. will see if a new variable is needed.

Reply:
renamed variable.